tutorials/i18n/pseudolocalization.rst

```diff
@@ -0,0 +1,82 @@
+
+Pseudolocalization can also be toggled at the runtime through GDScript (more on that 
+at the end of this tutorial).
+
+Pseudolocalization configurations
+---------------------------------
+
+Pseudolocalization in Godot can be set up according to the specific use case of the 
+project. Following are the pseudolocalization properties that can be configured through 
+project settings :
+
+-  ``replace_with_accents``, replaces all characters in the string with their accented 
+    variants. "The quick brown fox jumped over the lazy dog" will be converted to 
+    "Ŧh̀é q́üíćḱ ḅŕôŵή f́ôx́ ǰüm̀ṕéd́ ôṽéŕ ŧh̀é łáźý d́ôǵ" when this setting is enabled. 
+-  ``double_vowels``, doubles all the vowels in the string. It is a pretty good approximation
+    to simulate expansion of text during localization.
+-  ``fake_bidi``, or fake bidirectional text, simulates right-to-left text. This is useful to
+    simulate right-to-left writing systems.
+-  ``override``, replaces all the characters in the string with a '*'. This is useful for 
+    quickly finding text that isn't being localized.
+-  ``expansion_ratio``, can be used in cases where doubling the vowels isn't a sufficient 
+    approximation. This setting pads the string with '_' and expands it by the given ratio.
+    An expansion ratio of ``0.3`` is sufficient for most practical cases and it will increase 
+    the length of the string by 30%.
+-  ``prefix`` and ``suffix``, properties can be used to specify a prefix and suffix to wrap
+    the text in.
+-  ``skip_placeholders``, skips placeholders for string formatting like ``%s`` and ``%f``. 
+    This is useful to identify places where more arguments are required for the formatted string
+    to display correctly.
+
+All of these properties can be toggled as needed according to the project's use case.
+
+Configuring pseudolocalization at the run-time
+----------------------------------------------
+
+Pseudolocalization can be toggled at run-time using 
+:ref:`TranslationServer.set_pseudolocalization_enabled(enabled) <class_TranslationServer_method_set_pseudolocalization_enabled>`. 
+However, if run-time configuration of pseudolocalization properties is required, 
+they can be directly configured using
+:ref:`ProjectSettings.set(property, value) <class_ProjectSettings_method_set>`
+and then calling
+:ref:`TranslationServer.reload_pseudolocalization() <class_TranslationServer_method_reload_pseudolocalization>`
+which reparses the pseudolocalization properties and reloads the pseudolocalization. 
+Following code snippet shall turn on ``replace_with_accents`` and ``double_vowels`` properties and 
+then call ``reload_pseudolocalization()`` for the changes to get reflected :
+
+::
+
+    ProjectSettings.set("internationalization/pseudolocalization/replace_with_accents", true)
+    ProjectSettings.set("internationalization/pseudolocalization/double_vowels", true)
+	TranslationServer.reload_pseudolocalization()
+
+
```

Comment:
Use shorthand syntax for code blocks, and only add 1 newline at the end of file:

```suggestion
The following code snippet shall turn on ``replace_with_accents`` and ``double_vowels`` properties
and then call ``reload_pseudolocalization()`` for the changes to get reflected::

    ProjectSettings.set("internationalization/pseudolocalization/replace_with_accents", true)
    ProjectSettings.set("internationalization/pseudolocalization/double_vowels", true)
	TranslationServer.reload_pseudolocalization()

```